src/Credentials/GCECredentials.php

```diff
@@ -619,4 +631,9 @@ public function setIsOnGce($isOnGce)
         // Set isOnGce
         $this->isOnGce = $isOnGce;
     }
+
+    public function getCredType(): string
```

Comment:
Why did we decide to make this function public? It seems like there's no need to make the visibility public, and we could keep it simply `protected`, based on how it's being used.

Reply:
I should have been protected. Somehow missed this : )

Fixed.